frontend/src/components/Main/DatasetModelOptionList/DatasetModelOption.js

```diff
@@ -30,35 +31,25 @@ class DatasetModelOption extends Component {
                 </td>
                 <td>{dataset.metadata.name}</td>
                 <td>
-                    <select
-                        className="form-control"
+                    <SelectInput
+                        choices={getDegreeOptions(dataset).map(option => {
+                            return {value: option.value, text: option.label};
+                        })}
+                        onChange={value => updateOption(datasetId, "degree", parseInt(value))}
                         value={option.degree}
-                        onChange={e => updateOption(datasetId, "degree", parseInt(e.target.value))}>
-                        {getDegreeOptions(dataset).map(item => (
-                            <option key={item.value} value={item.value}>
-                                {item.label}
-                            </option>
-                        ))}
-                    </select>
+                    />
                 </td>
                 {hasAdverseDirection ? (
                     <td>
-                        <select
-                            className="form-control"
+                        <SelectInput
+                            choices={adverseDirectionOptions.map(option => {
+                                return {value: option.value, text: option.label};
+                            })}
+                            onChange={value =>
+                                updateOption(datasetId, "adverse_direction", parseInt(value))
```

Comment:
2 things:
1. do we actually need to cast to int here? react may be smart, i'm not sure
2. if we do need to cast to int, let's add an optional `cast: PropTypes.func` to the Select Input; if used, the value could be `parseInt` for example. That way the select component can handle casting. But let's only do this if  1. above is true and we do need to cast

Reply:
no we dont need to cast to int. i changed to value because its not user input